### ISSUE TITLE
SVGO example outputs wrong path at build for SVG loaded via SCSS

### ISSUE THREAD

**Issue by drdogbot7** [swashata/wp-webpack-script#284]
Using the SVGO example from the documentation, I get the wrong paths in compiled CSS when referencing SVG files. Dev server works fine.

To make sure this wasn't just something dumb I was doing, I created a minimal theme based on the example theme and added an svg background to the body.

Files
```
wpackio-theme
 |
 +-- src
 |  |  
 |  +-- theme
 |  |  |
 |  |  + heart.svg
 |  |  + main.js
 |  |  + main.scss
```

`wpackio.project.js` (copied straight from documentation)
```JS
webpackConfig: (config, api) => {
	// Create a new config
	const newConfig = { ...config };
	// Extend the rules for some great svg experience
	newConfig.module.rules = [
		...newConfig.module.rules,
		// SVGO Loader
		// https://github.com/rpominov/svgo-loader
		{
			test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
			use: [
				{ loader: 'file-loader' },
				{
					loader: 'svgo-loader',
					options: {
						plugins: [
							{ removeTitle: true },
							{ convertColors: { shorthex: false } },
							{ convertPathData: false },
						],
					},
				},
			],
		},
	];
	// Return it
	return newConfig;
},
```

`main.scss`
```SCSS
body {
  background-image:url('heart.svg');
}
```

CSS output using dev server: (THIS WORKS!)
```CSS
body {
  background-image: url(//localhost:3000/wp-content/themes/wpackio-theme/dist/5e1d0848e37267eb1456fa12bee167f9.svg);
}
```

CSS output using build (PATH IS WRONG)
```CSS
body {
  background-image: url(5e1d0848e37267eb1456fa12bee167f9.svg);
}
```

**Comment by swashata** [comment]
Hello,

Thanks for reporting. Can I ask you to share a github repo with min code to reproduce the error?

**Comment by drdogbot7** [comment]
This shows the issue for me.

https://github.com/drdogbot7/wpackio-issue-284

localhost:3000 after running `yarn start`
![image](https://user-images.githubusercontent.com/1050936/53830374-e5ac1a00-3f47-11e9-9f13-e0da43077423.png)

![image](https://user-images.githubusercontent.com/1050936/53830427-05434280-3f48-11e9-81a4-c995ff5346f5.png)


localhost:8080 after running `yarn build`
![image](https://user-images.githubusercontent.com/1050936/53830301-beede380-3f47-11e9-85e9-33e1d0d96597.png)

![image](https://user-images.githubusercontent.com/1050936/53830539-4fc4bf00-3f48-11e9-9bb0-2bbc1d375274.png)

**Comment by drdogbot7** [comment]
Any updates on this?

**Comment by swashata** [comment]
Hi,

Getting there 😄

**Comment by swashata** [comment]
Hi @drdogbot7,

Wanted to update you that I have found the cause of the issue. While it is possible to solve entirely from `wpackio.project.js` file, I think I can provide an API for easy `file-loader` options. Please stay tuned.

**Comment by swashata** [comment]
## Pending work since commit before this comment

- [ ] Update Docs for SVG guide.
- [ ] Update Docs for the function signature of `files[..].webpackConfig`.

**Comment by swashata** [comment]
Hey @drdogbot7,

I have released new version 3.0.0 and have updated the [docs for svgo](https://wpack.io/tutorials/using-various-svg-loader/#svgo-loader)

Do give this a try and if you are successful, feel free to close this.

**Comment by drdogbot7** [comment]
Yup. That fixes the issue. 👍
